integration_test/third_party_apps_test/applications/aerospike/enable

```diff
@@ -2,6 +2,14 @@
 
 set -e
 
+# Create a back up of the existing file so existing configurations are not lost.
```

Comment:
Remove all the places where this comment was copied since we don't do that here.